src/plugins.py

```diff
@@ -0,0 +1,28 @@
+# Plugin loader
+#
+# Modular plugin loader
+import importlib
+
+preloaded_plugins = []
+
+
+def load(config):
+    plugins_available = config["plugins"]
+
+    for plugin in plugins_available:
+        plugin_key = list(plugin.keys())[0]
+        plugin_data = plugin[plugin_key]
+        print(plugin_data)
```

Comment:
Let's maybe cut down on the prints. Also could we add a prefix to the print? Such as `[INFO]`, `[WARN]`, `[ERR]`?